### ISSUE TITLE
CompatCore is null on version 1.5.1

### ISSUE THREAD

**Issue by katubug** [CyberRat7/Extra_Compat#12]
Hi! I just updated to 1.5.1 from 1.5.0 and freeze up when entering a world. Part of the error reads:

```
[16:19:14] [Render thread/ERROR]: Exception caught during firing event: Cannot invoke "net.cyber_rat.extra_compat.core.CompatCore.addCreative(net.minecraftforge.event.BuildCreativeModeTabContentsEvent)" because "compatCore" is null
	Index: 1
```

Here is my full latest.log: https://gist.github.com/katubug/78e8ab65086708c74fb06acecfc0649c

**Comment by yurei-dll** [comment]
I'm getting the same error. It seems like it only happens when I open the creative inventory, from my testing.

https://gist.github.com/yurei-dll/4d2b02d197b6b2d4a2f9651d836e4c26

**Comment by MehVahdJukaar** [comment]
https://github.com/CyberRat7/Extra_Compat/blob/b2926e70dd3855457d704b05ad64d27716593c76/src/main/java/net/cyber_rat/extra_compat/ExtraCompat.java#L27 
Error from this line. Its not server safe

**Comment by Veynam** [comment]
same issue

**Comment by CyberRat7** [comment]
we are aware the newest version has many issues and are working on a fix please use an older version of the mod

**Comment by CyberRat7** [comment]
Should be fixed in 1.5.3 make sure you are using Temporal Api 1.6.6
